modules/campaignion_donation_type/campaignion_donation_type.info

```diff
@@ -26,8 +26,10 @@ dependencies[] = node
 dependencies[] = node_reference
 dependencies[] = opengraph_meta_upload
 dependencies[] = payment
+dependencies[] = pgbar
 dependencies[] = pgbar:pgbar
 dependencies[] = pgbar:pgbar_webform
```

Comment:
Does it really need a dependency on `pgbar`, `pgbar:pgbar` and `pbar:pgbar_webform`? Surly the latter depend on pgbar already?!

Reply:
Ah that was accidental, need to remove that. Thanks.

Reply:
Updated the commit to exclude the additional dependency.